dev-cpp/notcurses/notcurses-2.2.3.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS="cmake"
+inherit cmake-multilib
+
+DESCRIPTION="Blingful TUIs and character graphics"
+HOMEPAGE="https://notcurses.com"
+SRC_URI="https://github.com/dankamongmen/notcurses/archive/v${PV}.tar.gz -> ${P}.tar.gz https://github.com/dankamongmen/notcurses/releases/download/v${PV}/notcurses-doc-${PV}.tar.gz"
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+DEPEND="dev-libs/libunistring
+	media-video/ffmpeg
+	>=sys-libs/readline-8.0"
+RDEPEND="${DEPEND}"
+BDEPEND="doc? ( app-text/pandoc )"
```

Comment:
In other words, you already have the pre-generated documentation available, thus we don't need to depend on *pandoc* anymore right?

Reply:
yep, i'll make that change, thanks.

Reply:
I've removed the `BDEPEND` and also the conditional `-DUSE_PANDOC=on`. Done.